taskcluster/kinds/all-pr-inference/kind.yml

```diff
@@ -0,0 +1,36 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
You can add this if you like, but it's probably not necessary seeing as you only have one kind. Instead, I would suggest you add `run-on-tasks-for` to the `task-defaults` for the `inference` kind, which will have the same effect of causing these tasks to run in PRs without the dummy task.

You may also want to consider putting `github-push` in the `run-on-tasks-for` list so that the tests run on pushes as well.

The reason we have `all` and `all-pr` for the pipeline tasks is mainly to give us a single task to target that means "run the whole pipeline" (otherwise we'd need to specify `export` and a bunch of `evaluate` steps as `target-phase` whenever we kick things off).

Reply:
I've removed this file, leaving only the `inference` kind that has `run-on-tasks-for` defined.